tests/components/climate/test_trigger.py

```diff
@@ -65,54 +58,13 @@ async def setup_automation(
 
 
 @pytest.mark.parametrize(
-    ("trigger_target_config", "entity_id", "climates_in_target"),
-    [
-        (
-            {CONF_ENTITY_ID: "climate.standalone_climate"},
-            "climate.standalone_climate",
-            1,
-        ),
-        ({ATTR_LABEL_ID: "test_label"}, "climate.label_climate", 2),
-        ({ATTR_AREA_ID: "test_area"}, "climate.area_climate", 2),
-        ({ATTR_FLOOR_ID: "test_floor"}, "climate.area_climate", 2),
-        ({ATTR_LABEL_ID: "test_label"}, "climate.device_climate", 2),
-        ({ATTR_AREA_ID: "test_area"}, "climate.device_climate", 2),
-        ({ATTR_FLOOR_ID: "test_floor"}, "climate.device_climate", 2),
-        ({ATTR_DEVICE_ID: "test_device"}, "climate.device_climate", 1),
-    ],
+    ("trigger_target_config", "entity_id", "entities_in_target"),
```

Comment:
The helper could return the names too, but I think that makes it very difficult to read the test

Reply:
Yes, I agree. This is more readable.